sweepbatcher/sweep_batcher.go

```diff
@@ -184,17 +184,33 @@ func defaultPublishErrorLogger(err error, errMsg string, log btclog.Logger) {
 	log.Warnf("%s: %v", errMsg, err)
 }
 
-// SweepRequest is a request to sweep a specific outpoint.
+// Input specifies an UTXO with amount that is added to the batcher.
+type Input struct {
+	// Outpoint is the outpoint that is being swept.
+	Outpoint wire.OutPoint
+
+	// Value is the value of the outpoint that is being swept.
+	Value btcutil.Amount
+}
+
+// SweepRequest is a request to sweep an outpoint or a group of outpoints.
 type SweepRequest struct {
 	// SwapHash is the hash of the swap that is being swept.
 	SwapHash lntypes.Hash
 
 	// Outpoint is the outpoint that is being swept.
+	// This is only set when Group is empty.
 	Outpoint wire.OutPoint
 
 	// Value is the value of the outpoint that is being swept.
+	// This is only set when Group is empty.
 	Value btcutil.Amount
 
+	// Group specifies multiple inputs in the same request. All the inputs
+	// belong to the same swap and are added to the same batch. If it is
+	// specified, OutPoint and Value must not be specified.
+	Group []Input
```

Comment:
Could we just have the group? We don't really need the `Output` and `Value` then.

Reply:
Fixed. Thanks!
Also renamed the field from `Group` to `Inputs`.